src/cpp/shared/scripting/callback.cpp

```diff
@@ -17,9 +18,4 @@ void ironfist_lua_push(std::string arg) {
 }
 
 void ironfist_lua_pushmulti() {
-};
-
-void ScriptCallback(const char * funcName) {
```

Comment:
Have you tested using ScriptCallback with only funcName (without args) after removing this? I vaguely recall something wasn't working for me until I added this one